rules/falco_rules.yaml

```diff
@@ -329,20 +346,117 @@
   priority: NOTICE
   tags: [network, mitre_remote_service]
 
-- list: user_context_files
-  items: [.bashrc, .bash_profile]
+# These rules and supporting macros are more of an example for how to
+# use the fd.*ip and fd.*ip.name fields to match connection
+# information against ips, netmasks, and complete domain names.
+#
+# To use this rule, you should modify consider_all_outbound_conns and
+# populate allowed_{source,destination}_{ipaddrs,networks,domains} with the
+# values that make sense for your environment.
+- macro: consider_all_outbound_conns
+  condition: (never_true)
+
+# Note that this can be either individual IPs or netmasks
+- list: allowed_destination_ipaddrs
+  items: ['"127.0.0.1"', '"8.8.8.8"']
 
-- rule: Modify User Context
-  desc: Detect attempt to modify .bashrc file or .bash_profile file
+- list: allowed_destination_networks
+  items: ['"127.0.0.1/8"']
+
+- list: allowed_destination_domains
+  items: [google.com, www.yahoo.com]
+
+- rule: Unexpected outbound connection destination
+  desc: Detect any outbound connection to a destination outside of an allowed set of ips, networks, or domain names
   condition: >
-    open_write and fd.filename in (user_context_files)
+    consider_all_outbound_conns and outbound and not
+    ((fd.sip in (allowed_destination_ipaddrs)) or
+     (fd.snet in (allowed_destination_networks)) or
+     (fd.sip.name in (allowed_destination_domains)))
+  output: Disallowed outbound connection destination (command=%proc.cmdline connection=%fd.name user=%user.name)
```

Comment:
should we put both `sip` and `cip` in the output as we are showing the connection detail

Reply:
%fd.name has both halves, here's an example: 
```15:48:15.536202984: Notice Disallowed outbound connection destination (command=curl http://www.github.com connection=10.0.2.15:35873->192.30.255.113:80 user=mstemm)```

Reply:
good!